catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -58,6 +61,11 @@
      */
     private Set<String> namespacesAdded = new HashSet<>();
 
+    /*
+     * Stack of scoped namespace URI to prefix mappings
+     */
+    private Deque<Map<String, String>> scopeOfNamespacesAdded = new ArrayDeque<>();
```

Comment:
This class isn't thread safe. Is that ok? If so, you may want to add a note in the class' JavaDoc comments.